### PR TITLE
issue 3206 fix

### PR DESCRIPTION
**Pull request by PrakashDurlabhji**

### CODE REVIEW

**Comment by PrakashDurlabhji** [comment]
@maxceem I will complete it

**Comment by PrakashDurlabhji** [comment]
@maxceem PR updated and it will be shown below dropdown "Project is active".

Kindly provide feedback if any because current behaviour is that when "cancelled" is clicked then dropown list disappears and dialog appears. so I am not keeping dropdown list visible. It will be hidden as current behaviour. just dialog will appear at red line mentioned by you in above comment

**Comment by PrakashDurlabhji** [comment]
@maxceem updating

**Comment by PrakashDurlabhji** [comment]
@maxceem updated

**Comment by maxceem** [comment]
@PrakashDurlabhji I see 2 your comments https://github.com/appirio-tech/connect-app/pull/3307#issuecomment-533915853 and https://github.com/appirio-tech/connect-app/pull/3307#issuecomment-533915853 that this PR has been updated, but there are no new commits.

Would you be able to fix this PR as per my comment above https://github.com/appirio-tech/connect-app/pull/3307#pullrequestreview-291498937 or should I open this issue for pick up?

**Comment by PrakashDurlabhji** [comment]
@maxceem code update done and working

File: src/components/ProjectStatus/ProjectStatus.scss [hunk 1]
```diff
@@ -45,6 +45,13 @@
       }
     }
 
+    .dropdown-wrap {
+      .status-header {
+        display: flex;
+        justify-content: center;
+      }
+    }
+
```
Comment (maxceem) [inline src/components/ProjectStatus/ProjectStatus.scss]: @PrakashDurlabhji do we really need this change?

File: src/components/ProjectStatus/ProjectStatusChangeConfirmation.scss [hunk 2]
```diff
@@ -35,11 +35,6 @@
         width: 100%;
       }
     }
-  
-    &.dropdown-up {
-      top: auto;
-      bottom: 320px;
-    }
```
Comment (maxceem) [inline src/components/ProjectStatus/ProjectStatusChangeConfirmation.scss]: @PrakashDurlabhji do we really need this change?

File: src/projects/detail/containers/ProjectInfoContainer.scss [hunk 3]
```diff
@@ -59,6 +59,7 @@
   justify-content: center;
   margin-top: 20px;
   padding: 5px 0;
+  position: relative;
```
Comment (maxceem) [inline src/projects/detail/containers/ProjectInfoContainer.scss]: @PrakashDurlabhji do we really need this change?

**Comment by PrakashDurlabhji** [comment]
@maxceem PR update done

File: src/components/ProjectStatus/ProjectStatus.scss (hunk 1 above)
Comment (maxceem) [reply src/components/ProjectStatus/ProjectStatus.scss]: @PrakashDurlabhji please, let me know if this required change, of we can skip it?

File: src/components/ProjectStatus/ProjectStatusChangeConfirmation.scss (hunk 2 above)
Comment (maxceem) [reply src/components/ProjectStatus/ProjectStatusChangeConfirmation.scss]: @PrakashDurlabhji please, let me know if this required change, of we can skip it?

**Comment by maxceem** [comment]
@PrakashDurlabhji there are two changes which I'm not sure if they are required or no. Could you please have a look on the comment above https://github.com/appirio-tech/connect-app/pull/3307#pullrequestreview-293187084

**Comment by maxceem** [comment]
@PrakashDurlabhji do you have update here?

**Comment by PrakashDurlabhji** [comment]
@maxceem change 1 required to keep it center.
change 2 is not required hence skipped it

**Comment by PrakashDurlabhji** [comment]
@maxceem ready

**Pull request merged**